test-d/opaque.ts

```diff
@@ -113,6 +113,34 @@ const unwrapped2 = 123 as PlainValueUnwrapTagged;
 expectType<number>(unwrapped1);
 expectType<number>(unwrapped2);
 
+// UnwrapTagged/UnwrapOpaque should work on types with multiple tags.
+const unwrapped3 = '' as UnwrapTagged<NormalizedAbsolutePath>;
+const unwrapped4 = '' as UnwrapOpaque<NormalizedAbsolutePath>;
+expectType<string>(unwrapped3);
+expectType<string>(unwrapped4);
```

Comment:
these are new tests that i added because i realized that i accidentally broke `UnwrapTagged` without them in an earlier version of this PR